README.md

```diff
@@ -1 +1,177 @@
-# https://www.kaggle.com/c/h-and-m-personalized-fashion-recommendations
\ No newline at end of file
+# **laplace**, a Graph Neural Network based Recommendation Engine
+
+##### Table of Contents  
+- [Overview](#Overview)  
+- [Get Started](#Get_Started) 
+  - [Installation & Data](#Installation) 
+  - [Main Pipeline](#Main_Pipeline) 
+- [Advanced Usage](#Advanced_Usage)  
+    - [Hyperparameter tuning](#Hyperparameter_tuning)     
+- [Further todo](#Further_todo)  
+- [Recommended resources](#Recommended_resources)  
+
+<br>
+
+
+
+<a name="Overview"></a>
+
+## Overview
+
+**laplace** is an end-to-end ML framework to train and predict on neurally-enhanced graphs for recommendation. 
+
+The pipeline is designed for self-supervised edge prediction on heterogenous graphs. 
+
+<br>
```

Comment:
is `<br>` in markdown? https://github.com/applied-exploration/drift/commit/66bb2cce1b28eac662805a60a638fc711f3cd16c